rele/subscription.py

```diff
@@ -33,25 +41,32 @@ def set_prefix(self, prefix):
 
     @property
     def filter_by(self):
-        return self._filter_by
+        return self._filters
 
     def set_filter_by(self, filter_by):
-        self._filter_by = filter_by
+        self._filters = filter_by
 
     def __call__(self, data, **kwargs):
         if "published_at" in kwargs:
             kwargs["published_at"] = float(kwargs["published_at"])
 
-        if self._filter_returns_false(kwargs):
+        if self._any_filter_returns_false(kwargs):
             return
 
         return self._func(data, **kwargs)
 
     def __str__(self):
         return f"{self.name} - {self._func.__name__}"
 
-    def _filter_returns_false(self, kwargs):
-        return self._filter_by and not self._filter_by(kwargs)
+    def _any_filter_returns_false(self, kwargs):
+        if not self._filters:
```

Comment:
Would this work instead?
```python
return not all(filter(kwargs) for filter in self._filters)
```

Reply:
Almost. Too good to be `True`.

Replacing the `for` loop.

Reply:
> Almost. Too good to be `True`.

:joy: 

An option to make it possible is for `_init_filters` to set `self._filters`  to empty list by default :smile: